docs/source/java/flight_sql_jdbc_driver.rst

```diff
@@ -0,0 +1,120 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+============================
+Arrow Flight SQL JDBC Driver
+============================
+
+The Flight SQL JDBC driver is a JDBC driver implementation that uses
+the :doc:`Flight SQL protocol <../format/FlightSql>` under the hood.
+This driver can be used with any database that implements Flight SQL.
+
+.. contents::
+
+Installation and Requirements
+=============================
+
+The driver is compatible with JDK 8+.  On JDK 9+, the following JVM
+parameter is required:
+
+.. code-block:: shell
+
+   java --add-opens=java.base/java.nio=ALL-UNNAMED ...
+
+To add a dependency via Maven, use a ``pom.xml`` like the following:
+
+.. code-block:: xml
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <project xmlns="http://maven.apache.org/POM/4.0.0"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+     <modelVersion>4.0.0</modelVersion>
+     <groupId>org.example</groupId>
+     <artifactId>demo</artifactId>
+     <version>1.0-SNAPSHOT</version>
+     <properties>
+       <arrow.version>10.0.0</arrow.version>
+     </properties>
+     <dependencies>
+       <dependency>
+         <groupId>org.apache.arrow</groupId>
+         <artifactId>flight-sql-jdbc-driver</artifactId>
+         <version>${arrow.version}</version>
+       </dependency>
+     </dependencies>
+   </project>
+
+Connecting to a Database
+========================
+
+The URI format is as follows::
+
+  jdbc:arrow-flight-sql://HOSTNAME:PORT[/?param1=val1&param2=val2&...]
```

Comment:
Probably just me but I find these URI format definitions to be a little cryptic. Maybe include an example string with a couple of parameters so people have something concrete to refer to?

Reply:
Thanks, good idea. I added an example with some prose to explain the parameters.